Allow banning members outside of a guild, Clean-up cmds.py

Dremixbotdev opened the pull request.

Dremixbotdev committed:
Add cogs

RedTeaDev commented:
Check github action: https://github.com/RedCokeDevelopment/Teapot.py/pull/47/checks?check_run_id=2723436851

Dremixbotdev commented:
Change ``bot`` to ``self.bot``

Dremixbotdev committed:
Multiple changes | Check description  

1. Renamed bot to self.bot
2. Added ``unban``
3. Allowed a user who is not in the server to be banned
4. Changed import order

That's all I remember lol

Dremixbotdev committed:
Fix unreferenced error

Dremixbotdev committed:
Update cmds.py

Dremixbotdev committed:
Update cmds.py
